js/views/buyWizardVw.js

```diff
@@ -222,37 +223,35 @@ module.exports = baseVw.extend({
   },
 
   showMaps: function(){
-    "use strict";
     this.$el.find('.js-buyWizardMap').removeClass('hide');
-    this.$el.find('.js-buyWizardMapPlaceHolder').removeClass('hide');
-    this.hideMap = false;
   },
 
   hideMaps: function(){
-    "use strict";
     this.$el.find('.js-buyWizardMap').addClass('hide');
-    this.$el.find('.js-buyWizardMapPlaceHolder').addClass('hide');
-    this.hideMap = true;
   },
 
   createNewAddress: function(){
-    "use strict";
     var self = this;
     this.$el.find('.js-buyWizardAddress').addClass('hide');
     this.$el.find('.js-buyWizardNewAddress').removeClass('hide');
     this.$el.find('#buyWizardNameInput').focus();
+    this.$buyWizardMap.find('iframe').addClass('blurMore');
```

Comment:
$buyWizardMap is just a wrapper for our map. It's is incredibly unlikely another iframe (other than a new map that we control) will find it's way in there.

Re: blur more. Yeah, either I should make an exception, or we should have some other way to style the map being in a transition state when advanced graphics are off... maybe making it black and white (but I imagine that filter would also slow down slo computers)?